sdk/openai/openai/test/public/utils/utils.ts

```diff
@@ -52,7 +52,7 @@ export async function withDeployments<T>(
   deploymentsInfo: DeploymentInfo[] = [],
   run: (model: string) => Promise<T>,
   validate: (result: T) => void,
-  modelsListToSkip?: ModelInfo[],
+  options: { modelsListToSkip?: ModelInfo[]; modelsListToRun?: ModelInfo[] } = {},
```

Comment:
I am curious, why are we adding back filtering in for whisper?

Reply:
For some certain deployments in the test resources (a few of the gpt-4 models), the first request receives a 500 response. That triggers the retry attempts, and even with the timeout, the Whisper test suite still takes a long time to finish. I've tried filtering those models with 500 error code response, and it will reduce the Whisper test suite time to 20 minutes. That is still a lot compared to run with just the whisper deployment, which only takes only 2 minutes.

Reply:
I wonder if we could adjust the retry input options to help fix this. If retries don't work with streams, would disable retrying make the situation better?

Reply:
That's a great point, I'll try out with retry option and update accordingly!